src/app/navbar/navbar.component.css

```diff
@@ -59,3 +59,7 @@ mat-menu {
 .icon-sm {
   width: 25%;
 }
+
+.material-icons.md-18 {
```

Comment:
where's this used?

Reply:
It wasn't being used anymore. Deleted now.